pkg/server/ca/ca.go

```diff
@@ -131,5 +140,20 @@ func (ca *serverCA) SignJWTSVID(ctx context.Context, jsr *node.JSR) (string, err
 	if err != nil {
 		return "", fmt.Errorf("unable to sign JWT-SVID: %v", err)
 	}
+
+	labels := []telemetry.Label{
+		{
+			Name:  "spiffe_id",
+			Value: jsr.SpiffeId,
+		},
+	}
+	for _, audience := range jsr.Audience {
+		labels = append(labels, telemetry.Label{
+			Name:  "audience",
```

Comment:
can we have multiple labels with the same name?

Reply:
Metrics package itself doesn't do any processing of labels outside of filtering... so i guess it would be up to the sink?

Reply:
Ok. suggest we test this stuff out with datadog or something similar before cutting a release with it